tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -108,7 +119,20 @@ def test_property_missing(nhsd_apim_proxy_url, property, pointer, correlation_id
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_data_null(nhsd_apim_proxy_url, property, pointer, correlation_id):
     """
-    .. py:function:: Test null data
+    .. py:function:: Scenario: A user submitting a request with an empty required attribute \
+        in the request body receives a 400 'Null Value' response
+
```

Comment:
Needs a list of what are required attributes

Reply:
Addressed in request_properties.rst, linked below given/when/then